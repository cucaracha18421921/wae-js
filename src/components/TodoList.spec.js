import React from 'react';
import {shallow} from "enzyme";
import TodoList from "../containers/VisibleTodoList";
import {Provider} from "react-redux";
import configureStore from 'redux-mock-store';

let wrapper;
const mockStore = configureStore([]);

beforeEach(() => {
  wrapper = shallow(
      <Provider store={mockStore({})}>
        <TodoList todos={[{id:0,text:'todo1'}]} />
      </Provider>
    );
});

describe('Filters rendering', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
