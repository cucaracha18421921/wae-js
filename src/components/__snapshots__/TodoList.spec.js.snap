// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filters rendering renders correctly 1`] = `ShallowWrapper {}`;
